jobs/acceptance-tests/templates/run.erb

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+<% if p("bpm.enabled") %>
+exec /var/vcap/jobs/bpm/bin/bpm run smoke_tests
```

Comment:
This is wrong.

Reply:
Fixed in 294f869e844409006b8de0f79291e1e904ecb3ed ; though I notice that things still magically worked, _somehow_, even though `bpm.enabled` is true…